modules/claims_api/app/services/claims_api/evidence_waiver_document_service.rb

```diff
@@ -7,13 +7,24 @@ class EvidenceWaiverDocumentService < DocumentServiceBase
 
     def create_upload(claim:, pdf_path:, doc_type:, ptcpnt_vet_id:)
       unless File.exist? pdf_path
-        ClaimsApi::Logger.log(LOG_TAG, detail: "Error creating upload doc: #{pdf_path} doesn't exist,
-                                                    claim_id: #{claim.claim_id}")
+        ClaimsApi::Logger.log(LOG_TAG, ews_id: claim.id, claim_id: claim.claim_id,
+                                       detail: "Error creating upload doc: #{pdf_path} doesn't exist.")
         raise Errno::ENOENT, pdf_path
       end
 
+      ClaimsApi::Logger.log(LOG_TAG, ews_id: claim.id, claim_id: claim.claim_id,
+                                     detail: 'Starting document upload to BD.')
+
       body = generate_body(claim:, doc_type:, pdf_path:, ptcpnt_vet_id:)
-      ClaimsApi::BD.new.upload_document(identifier: claim.claim_id, doc_type_name: '5103', body:)
+      result = ClaimsApi::BD.new.upload_document(identifier: claim.claim_id, doc_type_name: FORM_SUFFIX, body:)
+
+      ClaimsApi::Logger.log(LOG_TAG, ews_id: claim.id, claim_id: claim.claim_id,
+                                     detail: 'Document upload to BD successful.')
+      result
+    rescue => e
```

Comment:
Noting here that we will log twice whenever the PDF path doesn't exist and raises on line 12. Likely fine - a bad pdf path is still a failure, but it is technically one we already knew about & handled.